test/routes/graphql/Query/organizations.test.ts

```
@@ -0,0 +1,137 @@
+	const mockContext = {
+		drizzleClient: {
+			query: {
+				organizationsTable: {
+					findFirst: vi.fn(),
+					findMany: vi.fn(),
+				},
+			},
+		},
+		log: { error: vi.fn(), info: vi.fn(), warn: vi.fn(), debug: vi.fn() },
+	};
+	return mockContext as unknown as GraphQLContext;;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)